lib/routes/bilibili/api-interface.d.ts

```diff
@@ -0,0 +1,384 @@
+    desc: string;
+    id: string;
+    jump_url: string;
+    label: string;
+    sketch_id: string;
+    style: number;
+    title: string;
+}
+interface Archive {
+    aid: string;
+    badge: Badge;
+    bvid: string;
+    cover: string;
+    desc: string;
+    disable_preview: number;
+    duration_text: string;
+    jump_url: string;
+    stat: Stat;
+    title: string;
+    type: number;
+}
+interface Major {
+    opus?: Opus;
+    type: MajorType;
+    archive?: Archive;
+    pgc?: Pgc;
+    live?: Live;
+    common?: Common2;
+}
+interface Moduledynamic {
+    additional?: Additional;
+    desc?: Desc;
+    major?: Major;
+    topic?: any;
+}
+interface Emoji {
+    icon_url: string;
+    size: number;
+    text: string;
+    type: number;
+}
+
+interface Item {
+    desc: Desc;
+    type: number;
+}
+interface Moduleinteraction {
+    items: Item[];
+}
+interface Threepointitem {
+    label: string;
+    type: string;
+}
+interface Modulemore {
+    three_point_items: Threepointitem[];
+}
+interface Comment {
+    count: number;
+    forbidden: boolean;
+}
+interface Like {
+    count: number;
+    forbidden: boolean;
+    status: boolean;
+}
+interface Modulestat {
+    comment: Comment;
+    forward: Comment;
+    like: Like;
+}
+interface Moduletag {
+    text: string;
+}
+interface Modules {
+    module_author: Moduleauthor;
+    module_dynamic: Moduledynamic;
+    module_interaction?: Moduleinteraction;
+    module_more?: Modulemore;
+    module_stat?: Modulestat;
+    module_tag?: Moduletag;
+}
+
+export interface Orig {
+    basic: Basic;
+    id_str: string;
+    modules: Modules;
+    type: DynamicType;
+    visible: boolean;
+}
+export interface Item2 {
+    basic: Basic;
+    id_str: string;
+    modules: Modules;
+    type: DynamicType;
+    visible: boolean;
+    orig?: Orig;
+}
+export interface Data {
+    has_more: boolean;
+    items: Item2[];
+    offset: string;
+    update_baseline: string;
+    update_num: number;
+}
+// https://api.bilibili.com/x/polymer/web-dynamic/v1/feed/space
+export interface BilibiliWebDynamicResponse {
+    code: number;
+    message: string;
+    ttl: number;
+    data: Data;
+}
+
+/**
+ * 作者类型
+ * AUTHOR_TYPE_PGC：番剧
+ * AUTHOR_TYPE_NORMAL：用户
+ */
+export type AuthorType = 'AUTHOR_TYPE_PGC' | 'AUTHOR_TYPE_NORMAL';
+
+/**
+ *  动态类型
+ * DYNAMIC_TYPE_FORWARD：转发动态
+ * DYNAMIC_TYPE_AV：视频投稿
+ * DYNAMIC_TYPE_PGC_UNION：番剧
+ * DYNAMIC_TYPE_DRAW：图文
+ * DYNAMIC_TYPE_WORD：文本
+ * DYNAMIC_TYPE_ARTICLE：专栏
+ * DYNAMIC_TYPE_LIVE：直播
+ * DYNAMIC_TYPE_COMMON_SQUARE：普通正方形（专题页）
+ */
+export type DynamicType = 'DYNAMIC_TYPE_FORWARD' | 'DYNAMIC_TYPE_AV' | 'DYNAMIC_TYPE_PGC_UNION' | 'DYNAMIC_TYPE_DRAW' | 'DYNAMIC_TYPE_WORD' | 'DYNAMIC_TYPE_ARTICLE';
+
+/**
+ * 主要类型
+ * MAJOR_TYPE_ARCHIVE：视频
+ * MAJOR_TYPE_PGC：番剧
+ * MAJOR_TYPE_OPUS：图文混排
+ * MAJOR_TYPE_COMMON：普通
+ * MAJOR_TYPE_LIVE：直播
+ */
+export type MajorType = 'MAJOR_TYPE_ARCHIVE' | 'MAJOR_TYPE_PGC' | 'MAJOR_TYPE_OPUS' | 'MAJOR_TYPE_COMMON' | 'MAJOR_TYPE_LIVE';
```

Comment:
```suggestion
export type MajorType = 'MAJOR_TYPE_ARCHIVE' | 'MAJOR_TYPE_PGC' | 'MAJOR_TYPE_OPUS' | 'MAJOR_TYPE_COMMON' | 'MAJOR_TYPE_LIVE' | 'MAJOR_TYPE_LIVE_RCMD';
```

`MAJOR_TYPE_LIVE_RCMD` is missing, it's live status (only present if the stream is online).

Here is a list for these types: https://github.com/SocialSisterYi/bilibili-API-collect/blob/master/docs/dynamic/dynamic_enum.md